src/main/java/Duke.java

```diff
@@ -1,10 +1,297 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final int MAX_TASK = 100;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printHello();
+        runProgram();
+        printBye();
+    }
+
+    private static void runProgram() {
+        Task[] taskList = new Task[MAX_TASK];
+        Scanner in = new Scanner(System.in);
+
+        //Loop to receive response.
+        while (true){
+
+            String input = in.nextLine();
+
+            // BYE command
+            if(input.equalsIgnoreCase("bye")){
+                return;
+            }
```

Comment:
The if statement should have a spacing before the brackets (condition statement) like the while statement.
I noticed the same issue in several other places too.

Reply:
Yeap, thank you! I will fix them.